### ISSUE TITLE
Duplicate related terms

### ISSUE THREAD

**Issue by mikemorr** [mvysny/aedict#779]
In this screenshot, お見事 shows two related terms 見事 1 and 見事 2, but they appear to be identical. I wonder if this is the desired behavior? Perhaps it is a redundancy in the dictionary data?
![screenshot_20170718-222823](https://user-images.githubusercontent.com/2184268/28369479-be95d372-6c65-11e7-903d-f4e4f71f6db9.png)

**Comment by mvysny** [comment]
That's how JMDict lists them:
```
<entry>
<ent_seq>2235980</ent_seq>
<k_ele>
<keb>お見事</keb>
</k_ele>
<k_ele>
<keb>お美事</keb>
</k_ele>
<r_ele>
<reb>おみごと</reb>
</r_ele>
<info>
<audit>
<upd_date>2007-11-18</upd_date>
<upd_detl>Entry created</upd_detl>
</audit>
</info>
<sense>
<pos>&int;</pos>
<xref>見事・1</xref>
<xref>見事・2</xref>
<gloss>well done!</gloss>
<gloss>bravo</gloss>
</sense>
</entry>
```
Quoting the documentation of the `xref` element:

> This element is used to indicate a cross-reference to another
	entry with a similar or related meaning or sense. The content of
	this element is typically a keb or reb element in another entry. In some
	cases a keb will be followed by a reb and/or a sense number to provide
	a precise target for the cross-reference. Where this happens, a JIS
	"centre-dot" (0x2126) is placed between the components of the 
	cross-reference.

**Comment by mvysny** [comment]
So, the dot reference targets both first reading, and second reading. If there is some way how this can be improved, please let me know.

**Comment by mikemorr** [comment]
Here is a screenshot after tapping the related terms link. Maybe when the references point to the same dictionary entry, the entry could be listed only once, instead of twice as seen here?

![screenshot_20170719-215724](https://user-images.githubusercontent.com/2184268/28397232-5d6b9488-6cce-11e7-94aa-f20dd3e7dd83.png)

Alternatively, it would be neat to somehow highlight the specific sense or senses that are targeted (in this case I guess that would be 1:"splendid, magnificent..." and 2:"praiseworthy act..." but not 3:"utter (esp. defeat)...". But I'm not sure how exactly that would work.

**Comment by mvysny** [comment]
Unfortunately, it is currently impossible to target a particular kanji or particular reading in an entry. I will thus make sure that the entry will appear only once

**Comment by mvysny** [comment]
Fixed in Aedict 3.41
